src/automate.py

```diff
@@ -213,7 +215,7 @@ def run_parallel(runnables, arguments) :
             p.start()
     for proc, executable, runnable in jobs :
         proc.join()
-        results.append((executables, executable, proc.exitcode))
+        results.append((executables, executable, proc.exitcode, None, None))
```

Comment:
Why are we not getting the performance data `perf_data` when we run it parallel ?